### ISSUE TITLE
Confusing language about inclusive “or”?

### ISSUE THREAD

**Issue by lbeaufort** [oscarlevin/discrete-book#114]
Hi there! I’m really enjoying your online textbook, thank you for making it free online and providing helpful exercises and examples!

I came across the language 

> Note that for us, or is the inclusive or (and not the sometimes used exclusive or) meaning that P ∨ Q and P ∨ Q is in fact true when both 
P and Q are true.

Source: http://discrete.openmathbooks.org/dmoi3/sec_intro-statements.html#wKm

This language was a bit confusing to me, as I’m re-learning discrete math after 15 years. It might be less confusing to say: 

> Note that for us, or is the inclusive or (and not the sometimes used exclusive or) meaning that P ∨ Q and P ∨ Q is true ~in fact~ when both 
P and Q are true *and when either P or Q is true.*

If you agree, I’d be happy to put in a pull request. Thanks again!

**Comment by davidfarmer** [comment]
I think it is more clear to say it in the other order:

    P ∨ Q is in fact true when P is true, or when Q is true,
    or when both are true.

Maybe it would be good to go on to talk about what exclusive or
means (and give a symbol to it), and then start a new paragraph
with the discussion of "and"?

**Comment by oscarlevin** [comment]
Good idea @davidfarmer , and thanks @lbeaufort for the suggestion.  I'll clean this paragraph up as I work on the new edition.
